validator/keymanager/remote-web3signer/v1/requests.go

```diff
@@ -263,3 +271,65 @@ func GetSyncCommitteeContributionAndProofSignRequest(request *validatorpb.SignRe
 		ContributionAndProof: contribution,
 	}, nil
 }
+
+// GetBlockV2BellatrixSignRequest maps the request for signing type BLOCK_V2 BELLATRIX.
+func GetBlockV2BellatrixSignRequest(request *validatorpb.SignRequest, genesisValidatorsRoot []byte) (*BlockV2BellatrixSignRequest, error) {
+	if request == nil {
+		return nil, errors.New("nil sign request provided")
+	}
+	var b interfaces.BeaconBlock
+	switch request.Object.(type) {
+	case *validatorpb.SignRequest_BlindedBlockV3:
+		BlindedBlockV3, ok := request.Object.(*validatorpb.SignRequest_BlindedBlockV3)
+		if !ok {
+			return nil, errors.New("failed to cast request object to blinded block v3")
+		}
+		if BlindedBlockV3 == nil {
+			return nil, errors.New("invalid sign request: BlindedBlockV3 is nil")
+		}
+		beaconBlock, err := wrapper.WrappedBeaconBlock(BlindedBlockV3.BlindedBlockV3)
+		if err != nil {
+			return nil, err
+		}
+		b = beaconBlock
+	case *validatorpb.SignRequest_BlockV3:
+		BlockV2Bellatrix, ok := request.Object.(*validatorpb.SignRequest_BlockV3)
```

Comment:
Should be `blockV2Bellatrix` (and actually it's `V3` - again, this versioning is confusing, why not simply use `bellatrix`?)